contrib/auto-render/auto-render.js

```diff
@@ -1,6 +1,6 @@
 /* eslint no-console:0 */
-/* global katex */
 
+import katex from "katex";
```

Comment:
Won't this result in the bundle that's built for auto-render.js also containing all of KaTeX?